ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -60,3 +59,28 @@ EOF
     echo "${OUTBOUND_TYPE} is not supported yet" || exit 1
   fi
 fi
+
+set +u
+REQUIRED_OCP_VERSION="4.12"
+isOldVersion=true
+if [ -z ${OCP_VERSION} ] || [ "$(printf '%s\n' "${REQUIRED_OCP_VERSION}" "${OCP_VERSION}" | sort --version-sort | head -n1)" = "${REQUIRED_OCP_VERSION}" ]; then
```

Comment:
Where set the `OCP_VERSION` value?

Reply:
some yaml set OCP_VERSION, some use base_ref, I will deal with base_ref when no OCP_VERSION.